components/Navbar.js

```diff
@@ -80,12 +80,12 @@ function Navbar() {
 
         {/* other nav links */}
         <div className="hidden lg:flex items-center space-x-3 ">
-          <Link href="/register">
+          <Link href="/Signup">
             <a className="py-1 px-2 font-medium rounded border-2 border-black border-box">
               Register
             </a>
           </Link>
-          <Link href="/signin">
+          <Link href="/Signin">
```

Comment:
please keep this as `/signin` and rename the file files to lowercase only using `git mv Signin.js signin.js` then commit